docs/commands/rhoas_service-registry_rule.md

```diff
@@ -0,0 +1,35 @@
+## rhoas service-registry rule
+
+Manage rules in a Service Registry instance
+
+### Synopsis
+
+Manage validity and compatibility rules for a Service Registry instance and artifacts.
+Service Registry rules govern how registry content can evolve over time.
+
+Rules can be configured as global rules for entire Service Registry instances or specific artifacts.
+
+
+### Examples
+
+```
+## Enable global compatibility rule for a specific Service Registry instance
```

Comment:
I think that "specific registry instance" sounds confusing. I would use "globally for artifacts" as more precise form
```suggestion
## Enable global compatibility rule for a specific Service Registry instance
```

Reply:
The service registry docs uses the term `global rules`. Does "Enable global compatibility rules for artifacts of current service registry instance" sounds good?

Reply:
yes. global rules sound good